recipes/diive/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
```

Comment:
Please remove this file. Recipes in conda-forge should solely depend on packages in conda-forge.